data/json/mapgen/marina.json

```diff
@@ -0,0 +1,198 @@
+        "........................................................................................................................",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    ",
+        "                                                    ................                                                    "
+      ],
+      "terrain": {
+        "%": [
+          "t_tree",
+          "t_tree_walnut",
+          "t_tree_chestnut",
+          "t_tree_beech",
+          "t_tree_hazelnut",
+          "t_tree_cottonwood",
+          "t_tree_elm"
+        ],
+        " ": [
+          [ "t_grass", 20 ],
+          [ "t_grass_dead", 3 ],
+          [ "t_grass_tall", 5 ],
+          [ "t_grass_long", 3 ],
+          [ "t_dirt", 5 ],
+          [ "t_shrub", 2 ],
+          [ "t_tree", 1 ]
+        ],
+        "^": [ "t_shrub" ],
+        "~": [ "t_floor" ],
+        "&": [ "t_floor" ],
+        "`": [ [ "t_water_dp", 20 ], "t_water_sh" ],
```

Comment:
You probably want to just make this exclusively deep water. Given how boats currently interact with shallow water (unable to move under their own power unless they also have wheels), I think trying to dock a boat in there would be an exercise in frustration, dodging random shallow water. 

I also just think the slips for the boats would generally be uniformly deep (in terms of our contrast between deep and shallow in the game). If it's done for aesthetic purposes here, we might just want to investigate suggesting some tileset makers make some water variations.

Reply:
> If it's done for aesthetic purposes here

Yeah, that's the main reason.
I'll make all water tiles deep ones for now.